.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+language: ruby
```

Comment:
Why Ruby? I know everything is going to run inside the Docker container… so either we skip `language` (if Travis allow us to do so) or we declare Python, which is what we really use. Does that make sense?

Reply:
I really thought about replacing `ruby`. I just was used to it. But, nevermind, I'll change it.